python/msa/event.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum, auto
+
+
+class Topics(Enum):
+    EVENT_STACK_CLEARED = auto()
```

Comment:
```suggestion
    STACK_CLEARED = (auto(), 0)
```

I was thinking more something like this for the constant declarations.